src/asm/parser.y

```diff
@@ -1154,8 +1161,8 @@ export_list	: export_list_entry
 export_list_entry : scoped_id	{ sym_Export($1); }
 ;
 
-include		: T_POP_INCLUDE string {
-			fstk_RunInclude($2);
+include		: label T_POP_INCLUDE string endofline {
```

Comment:
Does it make sense to have a label declared on the same line as an `INCLIDE`?

Reply:
It was allowed before, and could be done for the same reason pret does `Label:: INCBIN "file.bin"` often (if the `INCLUDE`d file is some kind of resource that goes with `Label`).

Reply:
I guess.